platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java

```diff
@@ -310,10 +310,10 @@ public void scheduleTimeout(IDomTimeoutNode timeoutNode, Double seconds) {
 	}
 
 	@Override
-	public void makeDraggable(IDomNode draggedNode, IDomNode initNode) {
+	public void makeDraggable(IDomNode movedNode, IDomNode draggedNode, IDomNode limitingNode) {
```

Comment:
I find the renaming of the `draggedNode` and `initNode` confusing, maybe `draggedNode` and `dragInitiationNode`?

Reply:
Renamed stuff as discussed.